Template/Pages/Worship Services.html

```diff
@@ -1,10 +1,14 @@
 <!--
  Title: Worship Services
  -->
-<p>SUNDAY 10:30 AM</p>
-<p>-Worship</p>
-<p>-Children leave for Childrens' Church part way through the service.</p>
-<p>Our Sunday morning worship experience includes singing praise hymns, both old and new, time for corporate and personal prayer and inspiring sermons.</p>
+<p class="center">SUNDAY 10:30 AM</p>
```

Comment:
Casing again.

Also, “a.m.” is the standard spelling. Check the dictionary in your cupboard.

Reply:
The all caps was just for emphasis/formatting.